docs/src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+import SwipeableViews from 'react-swipeable-views';
+import Avatar from '@material-ui/core/Avatar';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import KeyboardArrowLeftRounded from '@material-ui/icons/KeyboardArrowLeftRounded';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+import SvgUnity from '../../icons/SvgUnity';
+
+const data = [
+  { title: '2M', metadata: 'Weekly downloads on npm' },
+  { title: '69k', metadata: 'Stars on GitHub' },
+  { title: '2k', metadata: 'Open-source contributors' },
+  { title: '13k', metadata: 'Followers on Twitter' },
+];
+
+const UserFeedback = ({
+  quote,
+  profile,
+}: {
+  quote: string;
+  profile: { avatar: string; name: string; role: string; company?: React.ReactElement };
+}) => {
+  return (
+    <Box>
+      <Typography variant="subtitle1" color="#fff" sx={{ mb: 2 }}>
+        {quote}
+      </Typography>
+      <Box sx={{ display: 'flex' }}>
+        <Avatar
+          src={profile.avatar}
+          alt={`Picture of ${profile.name}`}
+          sx={{ width: 60, height: 60, border: '1px solid', borderColor: '#fff' }}
+        />
+        <Box sx={{ ml: 2 }}>
+          <Typography color="#fff" fontWeight="bold" sx={{ mb: 1 }}>
+            {profile.name},{' '}
+            <Box component="span" sx={{ color: 'grey.500', fontWeight: 'regular' }}>
+              {profile.role}
+            </Box>
+          </Typography>
+          {profile.company}
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+const TESTIMONIALS = [
+  {
+    quote:
+      "“Material-UI offers a wide variety of high quality components that have allowed us to ship features faster. Material-UI has been used by more than a hundred engineers in our organization. What’s more, Material-UI's well architected customization system has allowed us to differentiate ourselves in the marketplace.”",
+    profile: {
+      avatar: 'https://avatars.githubusercontent.com/u/28296253?v=4',
+      name: 'Joona Rahko',
+      role: 'Staff Software Engineer',
+      company: <SvgUnity color="#007FFF" height={21} width={58} />,
```

Comment:
Why are we using the color as a direct hex here? Could we just use the blue-500?